adds top_of_stairs_landing()

davidtcalabrese opened the pull request:
- [ ] adds another function - top_of_stairs_landing()
- [ ] adds another way to lose - take aspirin tainted with cyanide
- [ ] imports time module to allow for pauses

davidtcalabrese committed:
adds top_of_stairs_landing()

EmbeddedMan commented on MinnetonkaGame.py:
```diff
@@ -41,12 +43,44 @@ def big_room():
     elif answer == "2":
-        # TODO: top_of_stairs_landing()
+        top_of_stairs_landing()
         game_over("TBD")
```
This game_over() line can be removed as we now branch off to top_of_stairs_landing().

davidtcalabrese replied:
removed and commited

EmbeddedMan commented on MinnetonkaGame.py:
```diff
@@ -41,12 +43,44 @@ def big_room():
+    answer = input(">")
+
+    if answer == "1":
+        game_over("That aspirin was tainted with cyanide. Too bad. ")
```
Damn, That's kinda dark. :-) We may need to re-work some of these endings so the overall theme of the game is consistent. But that can come later.

davidtcalabrese replied:
Yes I agree, I just didn't feel like building out a branch there.

EmbeddedMan commented on MinnetonkaGame.py:
```diff
@@ -41,12 +43,44 @@ def big_room():
+
+# at fridge
+def fridge():
+    pass
```
I'm not familiar with "pass". Is that just sort of a Python place holder for "pretend there's some code here but it does nothing"?

davidtcalabrese replied:
yes, exactly. We get an unresolved reference error if we call fridge and it's not defined yet.  It's used a lot for functions like I had it or classes you know you're going to need later but don't feel like defining yet. To avoid an error you'd just say: 

```
class foo:
   pass
```

davidtcalabrese committed:
removes placeholder game_over

game_over replace with invocation to top_of_stairs_landing()

davidtcalabrese committed:
replace cyanide ending with generic TBD

maybe when we have a new branch and don't want to work on it yet we can just end it like you were initially (game_over("TBD)")

Then we can still test the game and we have a reminder that this branch either needs to be built upon or we need an idea for a game_over reason.

EmbeddedMan reviewed (approved):
Cool. Good to go now.
